check_config_win.py

```diff
@@ -0,0 +1,64 @@
+'''
```

Comment:
Nit: Run this file through `black` and add it to the `lint-python` job in the CI to make sure that future changes maintain a standard format.

Reply:
um... I'm not really sure how to do that, can you help me on this / give me some direction?

Reply:
Sure! For the first step, you should install `black`:

```bash
$ python3 -m pip install --user black
```

Then, use it to format the file:

```bash
$ black check-config-win.py
```

For the second step, locate these lines in `.github/workflows/ci.yml`:

```yaml
      - name: lint
        run: |
          black --check ff2mpv.py
          flake8 ff2mpv.py
```

...and change them to:

```yaml
      - name: lint
        run: |
          black --check ff2mpv.py check-config-win.py
          flake8 ff2mpv.py check-config-win.py
```